source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -165,81 +220,152 @@ Table 3.p (below): List of Data_Type Values
        Discovery_Service\ |br|\ |lb|
        eJournal\ |br|\ |lb|
        Repository
-     - PR, DR, TR, IR\ |br|\ |lb|
-       PR_P1, DR_D1, TR_J1, TR_J2, TR_J3, TR_J4
+     - PR, DR, TR
 
    * - Multimedia
-     - Multimedia content such as audio, image, streaming audio, streaming video, and video.
+     - Multimedia content such as audio, image, streaming audio, streaming video, and video, that cannot be easily classified as a specific multimedia Data_Type.
      - Multimedia\ |br|\ |lb|
        Multimedia_Collection
-     - PR, DR, IR\ |br|\ |lb|
-       PR_P1, DR_D1, IR_M1
+     - PR, DR, IR
+
+   * - News_Item
+     - An article from a newspaper or magazine, or a news item available as a standalone content item not aggregated into a title, for example in an institutional repository.
+     - A&I_Database\ |br|\ |lb|
+       Aggregated_Full_Content\ |br|\ |lb|
+       Discovery_Service\ |br|\ |lb|
+       eJournal\ |br|\ |lb|
```

Comment:
These Host_Types are not required to provide IR, so they shouldn't be listed here.